x-pack/plugin/sql/src/main/antlr/SqlBase.g4

```diff
@@ -215,6 +215,7 @@ valueExpression
 primaryExpression
     : castExpression                                                                 #cast
     | extractExpression                                                              #extract
+    | builtinDateFunction                                                            #currentDateFunction
```

Comment:
Having a separate section for each method is just noise.
We should either define the method directly as we do with CAST or define all time related methods under `builtinDateTimeFunction`.

Reply:
I thought about it, as I saw your intention there with the `builtinDateTimeFunction`. But couldn't think of a way to do it in the parser as we don't have `CURRENT DATE` and `CURRENT TIMESTAMP` but `CURRENT_DATE` and `CURRENT_TIMESTAMP`. if we try to have a common prefix then the lexer will allow whitespace so things like `CURRENT_                    DATE`.